src/CoreBundle/Bridge/Chameleon/Module/NavigationTree/NavigationTree.php

```diff
@@ -394,7 +394,7 @@ protected function moveNode(): bool
         $parentNodeId = $this->inputFilterUtil->getFilteredGetInput('parentNodeId', '');
         $position = (int)($this->inputFilterUtil->getFilteredGetInput('position', '0'));
 
-        if ('' === $treeTableName || '' === $tableId || '' === $nodeId || '' === $parentNodeId || '' === $position) {
+        if ('' === $treeTableName || '' === $tableId || '' === $nodeId || '' === $parentNodeId) {
```

Comment:
I removed the check for `'' === $position` because `$position` can never be an empty string as it is cast to string. There even is a check further down checking for `0` and changing the behaviour in that case

Reply:
Re-examining this in the PR: A similar problem was solved in a different way (moving the  cast to after the `if`-statement). We should probably use the same solution to both of these